components/inputs/input-checkbox.js

```diff
@@ -210,7 +210,6 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(LitEle
 					aria-label="${ifDefined(this.ariaLabel)}"
 					@change="${this.#handleChange}"
 					class="d2l-input-checkbox"
-					@click="${this._handleClick}"
```

Comment:
Did this just not exist?

Reply:
That was my fault -- I [removed the method here](https://github.com/BrightspaceUI/core/pull/5409/files) and didn't realize this needed cleaning up.

Reply:
Forgot to comment on this - noticed this in my investigation and cleaned it up while I was here